apps/dashboard/src/components/OgEditor.tsx

```diff
@@ -181,6 +181,46 @@ export function OgEditor({ initialElements, localStorageKey: key, width, height
         return
       }
 
+      // Move down
+      if (event.key === 'ArrowDown' && selectedElement) {
+        event.preventDefault()
+        const element = elements.find(item => item.id === selectedElement)
+        updateElement({
+          ...element,
+          y: element.y + 1
```

Comment:
I forgot to write it in https://github.com/QuiiBz/ogstudio/issues/10, could we move by 10px if the `Shift` key is pressed, and `1px` otherwise?

Reply:
Yea, I wanted to make something like that, I'll make it works rn.